lib/exercises.rb

```diff
@@ -2,29 +2,98 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n), goes through the length of the string array
+# Space Complexity: O(n), size of the output hash depends on the size of the string array
 
 def grouped_anagrams(strings)
```

Comment:
Nice use of a hash as the key to another hash.

Given the strings are all pretty small this is O(n).  If they could be any length then this would be O(nm) where m is the length of the strings.